.travis.yml

```diff
@@ -224,17 +224,10 @@ jobs:
   - <<: *stage_deploy
     if: tag IS present
     <<: *python_3_7_mixture
-    install: skip
-    script: skip
-    deploy:
-      provider: pypi
-      on:
-        all_branches: true
-      user: jaraco
-      password:
-        secure: RAfz06AINvz7bfij/YhfkAreRqamgxS8a6jSRNxntYhtJke3ZszUbIDag8+n1I+G5XT2LnMhHqPNR7Plc+AeMz7VXTuy+b81Li5kse20NYlPhd7mBVmTUpXtqYQashV5J39F4qkATBLznCOrMEomM07VTXjO/o2hmQuXniab2Uo=
-      distributions: dists
-      skip_cleanup: true
+    install:
+    - python -m pip install tox tox-venv
```

Comment:
```suggestion
    - python -m pip install tox tox-venv
    - python -m tox --notest
```

Reply:
Thanks but no thanks. I'm not going to add a command and complication when the default process does the same thing. Better to rely on tox to do what it does best, rather than try to micro-manage it. This change seems counter to best-practices... and I can hardly imagine the benefit.

Reply:
I consider it best practice. Seen it used in a lot of places. tox itself uses `--notest` as it was designed for such separation: https://github.com/tox-dev/azure-pipelines-template/blob/master/run-tox-env.yml#L118